### PR TITLE
(PDB-1798) add jessie to build targets

### PR DESCRIPTION
**Pull request by wkalt**
This adds jessie to our build targets. We probably shouldn't merge it until we
get a nightly run with a green cell on jessie.

### CODE REVIEW

File: ext/jenkins/packaging.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ echo "**********************************************"
 
 set -x
 
-export COW="base-precise-amd64.cow base-trusty-amd64.cow base-wheezy-amd64.cow"
+export COW="base-precise-amd64.cow base-trusty-amd64.cow base-wheezy-amd64.cow base-jessie-amd64.cow"
```
Comment (melissa) [inline ext/jenkins/packaging.sh]: Want to put them in alphabetical order? (it helps my brain)

**Comment by melissa** [comment]
:+1:

**Comment by kbarber** [comment]
@wkalt isn't this chicken and egg? We can't get a green run on jessie, because there is no packaging for Jessie yet? At least I don't see a jessie build in the packages yet.

**Comment by kbarber** [comment]
I'm going to merge this, and re-run the nightly for stable from last night.

**Pull request merged**